design/ceph/node-loss-rbd-cephfs.md

```diff
@@ -0,0 +1,150 @@
+# Recover RBD/CephFS RWO PVC in case of Node Loss
+
+## Goal
+
+Faster RBD/CephFS RWO recovery in case of node loss.
+
+## Problem
+
+For RBD RWO recovery:
+
+When a node is lost where a pod is running with the RBD RWO volume is mounted, the volume cannot automatically be mounted on another node. If two clients are write to the same volume it could cause corruption. The node must be guaranteed to be down before the volume can be mounted on another node.
+
+For CephFS recovery:
+
+With the current design the node recovery will faster for CephFS.
+
+## Current Solution
+
+For RBD RWO recovery:
+
+We have a manual solution to the problem which involves forceful deletion of a pod so that forced detachment and attachment work is possible. The problem with the current solution is that even after the forced pod deletion it takes around 11 minutes for the volume to mount on the new node. Also there are still chances of data corruption if the old pod on the lost node comes back online, causing multiple writers and lead to data corruption. There is a another [documentation](../../Documentation/Troubleshooting/ceph-csi-common-issues.md#node-loss) which avoid data corruption which is baiscally if manual process of blocklisting NodeIPs.
```

Comment:
```suggestion
We have a manual solution to the problem which involves forceful deletion of a pod so that forced detachment and attachment work is possible. The problem with the current solution is that even after the forced pod deletion it takes around 11 minutes for the volume to mount on the new node. Also there are still chances of data corruption if the old pod on the lost node comes back online, causing multiple writers and lead to data corruption if the [documentation](../../Documentation/Troubleshooting/ceph-csi-common-issues.md#node-loss) is not followed to  manually block nodes.
```

Reply:
We may want to have a URL specifically to the 1.11 docs here since this doc will likely change with the new solution.

Reply:
added link for 1.11